### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by mumbi58**
# Description
I have created a homepage with minimal changes. The primary motivation was to establish a basic online presence. There are no specific dependencies required for this simple homepage setup.

#2 
## Type of change
- [ ] New feature (non-breaking change which adds functionality)
## Screenshot (optional)
image

Ensured proper rendering on various screen sizes by adjusting the browser window width in the inspect tool.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+}
+.dropdown i.fa-angle-down {
+  vertical-align: -3px; /* Adjust the value as needed to move the icon down */
+}
+
+
+
+
+
+
```
Comment (paulkahura) [inline src/css/style.css]: remove the spaces

File: src/css/style.css
Comment (paulkahura) [inline src/css/style.css:1]: use rem/em over px

File: src/css/style.css [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+  padding: 0;
+}
+
+.navcontainer li {
+  font-size: 16px;
+  color: black;
+  font-family: DM Sans, Regular;
+  margin-right: 20px;
+}
+
```
Comment (paulkahura) [inline src/css/style.css]: check the font and the size

**Review by paulkahura — CHANGES_REQUESTED** [review]
Hello there nice attempt but kindly include screenshots. Mobile and tablet. Good day!

**Comment by JAPHETHNYARANGA** [comment]
Thia is good work, ATleast you are learning. Closing this pr as it is already done

**Pull request closed without merging**